tests/remote.bats

```diff
@@ -35,3 +35,8 @@ load test-helper
   run "${clicmd}" remote commands json
   assert_success
 }
+
+@test "$clinom remote reverse" {
+  run "${clicmd}" remote reverse
+  assert_output --partial '"ip"' && assert_output --partial '"org"' && assert_output --partial '"country"' && assert_output --partial '"city"' && assert_output --partial '"postal"' && assert_output --partial '"timezone"'
```

Comment:
I don't know how I feel about all those conditionals if the test fails...
Is it possible to create a test that fails to see if it works correctly.

Reply:
thanks @rjpadilla ... this is just a better `assert_success`